src/model/DataLogger.h

```diff
@@ -1,9 +1,29 @@
 #pragma once
 #include <Arduino.h>
 
-// Logging intervals (1 sec, 1 min)
-#define SD_LOGGING_INTERVAL 1000
-#define SERIAL_LOGGING_INTERVAL 60000
+/*
+ * @brief DataLogger is a singleton that logs data to the SD card and serial port
+ *
+ * The singleton is called from the loop() method of the TankController class.
+ * It manages three logs: data, serial, and remote.
+ *
+ * The data log is written every second and includes the following:
+ * "time,tankid,temp,temp setpoint,pH,pH setpoint,onTime,Kp,Ki,Kd".
+ *
+ * The serial log is written every minute and includes the following:
+ * "HH:MM, current pH, current temperature".
+ *
+ * The remote log is written every minute and includes both data and other events.
+ * The other events are based on either the PHProbe reporting a new slope or the
+ * EEPROM having a new value written to it. In each case, it means a configuration
+ * changed.
+ *
+ */
+
+// Logging intervals (1 sec, 1 min, 1 min)
+const unsigned long DATA_LOGGING_INTERVAL = 1000;     // 1 sec logging of basic data to SD card
+const unsigned long REMOTE_LOGGING_INTERVAL = 60000;  // 1 min logging of data and other events
+const unsigned long SERIAL_LOGGING_INTERVAL = 60000;  // 1 min logging of data and debugging info
```

Comment:
What motivates your choice to put remote logging on a timer rather than to have it flagged whenever data is written to the log file? Wouldn't we want to upload notifications of EEPROM changes or extreme thermal/pH probe readings without delay?